src/services/dataSource.js

```diff
@@ -44,7 +44,9 @@ export const createDataSource = (data, context) => {
 };
 
 export const updateDataSourceSecurityRules = (dataSourceId, securityRules) => {
-  const accessRules = Object.assign(securityRules, { enabled: true });
+  const accessRules = securityRules.map(securityRule => {
```

Comment:
If the data source has rules that are meant to stay disabled, i.e. unrelated to the security rule updates, does this not mean that they all get enabled? We don't want unrelated security rules be enabled.

Reply:
@tonytlwu Yes, you are right. I'll come up with a solution that won't do that.